definitions.tex

```diff
@@ -12,5 +12,123 @@
 structures (e.g. ASTs, metadata, etc) or any intermediate representation chosen
 by the implementer.
 
+\indexdefn{module}
+\indexdefn{module|named}
+\definition{module, named module}{definitions.module}
+The collection of translation units whose \grammarterm{module-declaration}
+specify the same \grammarterm{module-name}.
+See \Cpp{} Standard [module.unit] paragraph 1.
+
+\indexdefn{module!unit}
+\definition{module unit}{definitions.module-unit}
+A translation unit that contains a \grammarterm{module-declaration}.
+See \Cpp{} Standard [module.unit] paragraph 1.
+
+\indexdefn{module!interface}
+\definition{module interface}{definitions.module-interface}
+The set of declarations belonging to a module that are necessarily reachable
+(\Cpp{} Standard [module.reach] paragraph 1) from outside of that module.
+See \Cpp{} Standard [module].
```

Comment:
From @Bigcheese on the 2020-05-22 telecon:

Expand this a little bit to be clear to an extended audience what this means.